.env.example

```
@@ -59,14 +59,30 @@ EMAIL_TO=username@gmail.com
+
+# Linear workspace ID (optional)
 LINEAR_WORKSPACE_ID=your-linear-workspace-id
-# Linear project ID
+
+# Linear project ID (optional)
 LINEAR_PROJECT_ID=your-linear-project-id
 
+# JSON Maps for Linear Parsing (optional, used by SmartParser for High Fidelity Parsing)
+# Format: {"obsidian_value": "linear_id", ...}
+# Example user map: {"sean": "user-id-123", "john": "user-id-456"}
+LINEAR_USER_MAP_JSON={"your_username": "linear-user-id"}
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)